src/arrayMethodSort.js

```diff
@@ -4,8 +4,32 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = () => {
+    for (let i = 0; i < this.length - 1; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        const a = '' + this[i];
+        const b = '' + this[j];
+        const change = this[i];
+
+        if (a > b) {
+          this[i] = this[j];
+          this[j] = change;
+        }
+      }
+    }
```

Comment:
```
      const defaultCompareFunction = (a, b) => {
        const aStr = a.toString();
        const bStr = b.toString();

        if (aStr < bStr) {
          return -1;
        }

        if (aStr > bStr) {
          return 1;
        }

        return 0;
      };
```

I meant something like this. And then we can set this defaultCompareFunction as a default value for our function or event easier we can do like below:
`compareFunction = (a, b) => a.toString() > b.toString()`